stock_location_orderpoint_cleanup/models/stock_location_orderpoint.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo import _, api, models
```

Comment:
To keep the dependency tree smaller, we can just use groupby from odoo, then base_partition is not neede. 

```suggestion
from odoo import _, api, models
from odoo.tools import groupby
```

Reply:
I don't like the usage of that function as it returns a list and argument for a field should use lambda. So, here, I use the field name and it returns directly a recordset.